packages/form-validations/src/FormValidations.ts

```diff
@@ -0,0 +1,496 @@
+import { lang } from './lang';
+import { tmpl, findAncestor, groupedElemCount, mergeConfig } from './utils';
+
+type Config = {
+  formFieldSelector: string;
+  errorClass: string;
+  successClass: string;
+  errorTextParentSelector: string;
+  errorTextTag: string;
+  errorTextClass: string;
+  dataElementMessage: string;
+};
+
+const defaultConfig: Config = {
+  formFieldSelector: '[data-spirit-validate]',
+  errorClass: 'has-error',
+  successClass: 'has-success',
+  errorTextParentSelector: '[data-spirit-validate]',
+  errorTextTag: 'div',
+  errorTextClass: '',
+  dataElementMessage: 'validation-text',
+};
+
+const VALIDATIONS_ERROR = 'form-validations-message';
+const SELECTOR = 'input:not([type^=hidden]):not([type^=submit]), select, textarea';
+const ALLOWED_ATTRIBUTES = ['required', 'min', 'max', 'minlength', 'maxlength', 'pattern'];
+// const EMAIL_REGEX = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/
```

Comment:
there is a lot of commented code, do we want to have it in the git history? I understand that you will add it later, but can we keep it clean now?

Reply:
I want to keep this because I am going to uncomment this.

Reply:
This goes on in https://github.com/lmc-eu/spirit-design-system/pull/826

Reply:
I will merge this together after that PR is checked.